platform/features/layout/res/templates/fixed.html

```diff
@@ -43,15 +48,15 @@
     </div>
 
     <!-- Selection highlight, handles -->
-    <span ng-if="controller.selected()">
-        <div class="l-fixed-position-item s-selectable s-selected s-moveable"
+    <span class="s-selected s-moveable" ng-if="controller.selected()">
```

Comment:
@larkin The check is there to show/hide markup for drag/resize in fixed position. The markup is different than the layout.